core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -36,9 +36,7 @@ angular.module('oppia').directive('ckEditor4Rte', [
         var _RICH_TEXT_COMPONENTS = RteHelperService.getRichTextComponents();
         var names = [];
         var icons = [];
-        var contextIsLessonRelated = (
-          ContextService.getPageContext() === PAGE_CONTEXT.TOPIC_EDITOR ||
-          ContextService.getPageContext() === PAGE_CONTEXT.SKILL_EDITOR);
+        var contextIsLessonRelated = ContextService.isContextLessonRelated();
```

Comment:
The (existing) naming of this is actually quite confusing because e.g. the learner exploration page is "lesson related" but is not included here. Maybe call it isContextTopicOrSkillEditor()? (Or is there some unifying concept that you want to convey here?)

Reply:
The unifying concept here would be whether this is, in any way, linked to a skill. 
In a skill editor and topic editor, this is direct. There is no need of this in the story editor, and in the exploration editor, if it is linked to a story, it means it is linked to a topic which implies it is related to some skills.
So, maybe isRelatedToSkills() or isLinkedToSkills()?

Reply:
canReferToSkills perhaps?

Reply:
Yeah, sg, will change this.

Reply:
Done